### PR TITLE
Add Test Plan Management page

### PR DESCRIPTION
**Pull request by howard-e**
This adds the initial foundation for the test plan management page. This is based on recent discussions to advance the Test Plan Reports' phase from Draft -> Candidate -> Recommended grouped by the ATs.
eg. Moving Alert Example to Candidate Phase for JAWS, NVDA & VoiceOver on macOS all at once.

How to run:
1. `yarn install`
2. Run the application and sign in as an admin
3. Navigate to `localhost:3000/test-management`

**Note: This builds onto #461. We can change the base branch back to `main` once that's merged**

### CODE REVIEW

**Review by evmiguel — CHANGES_REQUESTED** [review]
Well done, @howard-e! The disclosure component is slick and the new Test Management system is implemented very well. One blocker is the statuses, and it's probably a larger conversation around how we deal with the `Recommended` state. 

@s3ththompson could you chime in on whether we can use `In Review` as `Candidate` and `Finalized` as `Recommended`?

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js [hunk 1]
```diff
@@ -51,49 +57,31 @@ const updateStatusResolver = async (
...
+            const candidateStatusReachedAt = new Date();
+            updateParams = {
+                ...updateParams,
+                candidateStatusReachedAt,
+                metrics: { ...testPlanReport.metrics, ...metrics },
+                recommendedStatusTargetDate: recommendedStatusTargetDateResolver(
+                    {
+                        candidateStatusReachedAt
+                    }
+                )
```
Comment (evmiguel) [inline server/resolvers/TestPlanReportOperations/updateStatusResolver.js:75]: `vendorReviewStatus: 'READY'` should go here

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js [hunk 2]
```diff
@@ -51,49 +57,31 @@ const updateStatusResolver = async (
...
+            };
+        } else if (status === 'RECOMMENDED') {
+            updateParams = {
+                ...updateParams,
+                metrics: { ...testPlanReport.metrics, ...metrics },
+                recommendedStatusReachedAt: new Date(),
+                vendorReviewStatus: 'READY'
```
Comment (evmiguel) [inline server/resolvers/TestPlanReportOperations/updateStatusResolver.js:82]: Remove `vendorReviewStatus`

**Review by evmiguel — CHANGES_REQUESTED** [review]
New phase changes look good! Only one request, and it's good to go.

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js (hunk 1 above)
Comment (evmiguel) [reply server/resolvers/TestPlanReportOperations/updateStatusResolver.js:75]: The `vendorReviewStatus` gets set when the report goes to the Candidate phase. Once it's been recommended, the `vendorReviewStatus` should be approved.

Comment (evmiguel) [reply server/resolvers/TestPlanReportOperations/updateStatusResolver.js:75]:
```suggestion
                ),
                vendorReviewStatus: 'READY'
```

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js (hunk 2 above)
Comment (evmiguel) [reply server/resolvers/TestPlanReportOperations/updateStatusResolver.js:82]:
```suggestion
                vendorReviewStatus: 'APPROVED'
```

File: client/components/TestManagement/StatusSummaryRow/index.jsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        setShowThemedModal(true);
+    };
+
+    const onThemedModalClose = () => {
+        setShowThemedModal(false);
+        dropdownUpdateReportStatusButtonRef.current.focus();
+    };
+
+    let phase = 'Draft'; // also accounts for IN_REVIEW
+    if (testPlanReports.every(i => i.status === 'RECOMMENDED'))
```
Comment (evmiguel) [inline client/components/TestManagement/StatusSummaryRow/index.jsx:105]: Should this be `testPlanReports.every(i => i.status === 'RECOMMENDED') || testPlanReports.every(i => i.vendorReviewStatus === 'APPROVED')` ?

Or should the status of the report be moved to `RECOMMENDED` when the vendor approves the report?

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js (hunk 1 above)
Comment (howard-e) [reply server/resolvers/TestPlanReportOperations/updateStatusResolver.js:75]: Good catch!

File: server/resolvers/TestPlanReportOperations/updateStatusResolver.js (hunk 2 above)
Comment (howard-e) [reply server/resolvers/TestPlanReportOperations/updateStatusResolver.js]: Thanks!

File: client/components/ManageTestQueue/index.jsx [hunk 4]
```diff
@@ -556,290 +495,259 @@ const ManageTestQueue = ({
...
+                    title={feedbackModalTitle}
+                    content={feedbackModalContent}
+                    closeLabel="Ok"
+                    handleClose={() => {
+                        setShowFeedbackModal(false);
+                        focusButtonRef.current.focus();
+                    }}
+                />
+            )}
+            {/*</Container>*/}
```
Comment (evmiguel) [inline client/components/ManageTestQueue/index.jsx:750]:
```suggestion
```

**Review by evmiguel — APPROVED** [review]

**Pull request merged**